cms/settings/base.py

```diff
@@ -196,38 +197,51 @@
 #
 # Do not use the same Redis instance for other things like Celery!
 
-# Prefer the TLS connection URL over non
-REDIS_URL = env.get("REDIS_TLS_URL", env.get("REDIS_URL"))
+CACHES: dict = {"memory": {"BACKEND": "django.core.cache.backends.locmem.LocMemCache", "LOCATION": "memory"}}
 
-if REDIS_URL:
-    connection_pool_kwargs: dict[str, str | None] = {}
+redis_options = {
+    "IGNORE_EXCEPTIONS": True,
+    "SOCKET_CONNECT_TIMEOUT": 2,  # seconds
+    "SOCKET_TIMEOUT": 2,  # seconds
+}
+DJANGO_REDIS_LOG_IGNORED_EXCEPTIONS = True
+
+# Prefer the TLS connection URL over non for Heroku
+if redis_url := env.get("REDIS_TLS_URL", env.get("REDIS_URL")):
+    connection_pool_kwargs: dict = {}
 
-    if REDIS_URL.startswith("rediss"):
-        # Heroku Redis uses self-signed certificates for secure redis conections. https://stackoverflow.com/a/66286068
+    if redis_url.startswith("rediss"):
+        # Heroku Redis uses self-signed certificates for secure redis connections. https://stackoverflow.com/a/66286068
         # When using TLS, we need to disable certificate validation checks.
         connection_pool_kwargs["ssl_cert_reqs"] = None
 
-    redis_options = {
-        "IGNORE_EXCEPTIONS": True,
-        "SOCKET_CONNECT_TIMEOUT": 2,  # seconds
-        "SOCKET_TIMEOUT": 2,  # seconds
-        "CONNECTION_POOL_KWARGS": connection_pool_kwargs,
+    CACHES["default"] = {
+        "BACKEND": "django_redis.cache.RedisCache",
+        "LOCATION": redis_url,
+        "OPTIONS": {**redis_options, "CONNECTION_POOL_KWARGS": connection_pool_kwargs},
     }
 
-    CACHES = {
-        "default": {
-            "BACKEND": "django_redis.cache.RedisCache",
-            "LOCATION": REDIS_URL,
-            "OPTIONS": redis_options,
-        }
+elif elasticache_addr := env.get("ELASTICACHE_ADDR"):
+    port = env["ELASTICACHE_PORT"]
```

Comment:
nitpick: there is no checking for this and cluster name/region.
I know this won't be a problem per se as it will be diligently Terraformed, but still.

Reply:
Fair enough